kong/dao/migrations/cassandra.lua

```diff
@@ -748,5 +748,16 @@ return {
       CREATE INDEX IF NOT EXISTS ON consumers(custom_id);
       CREATE INDEX IF NOT EXISTS ON consumers(username);
     ]]
+  },
+  {
+    name = "2018-05-17-173100_hash_on_cookie",
```

Comment:
Is there some automated way contributors are generating these migrations? I just manually created the timestamp/ name and it seemed to work.